.github/workflows/tests.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
     needs: lint
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [ 3.8, 3.9, '3.10', '3.11']
```

Comment:
It looks like GitHub doesn't support Python 3.11 yet.

Reply:
fixed